allennlp/data/dataset_readers/conll2003.py

```diff
@@ -66,13 +67,19 @@ class Conll2003DatasetReader(DatasetReader):
         Each will have its own namespace : `pos_tags`, `chunk_tags`, `ner_tags`.
         If you want to use one of the tags as a `feature` in your model, it should be
         specified here.
-    coding_scheme : `str`, optional (default=`IOB1`)
+    convert_to_coding_scheme : `str`, optional (default=`None`)
         Specifies the coding scheme for `ner_labels` and `chunk_labels`.
-        Valid options are `IOB1` and `BIOUL`.  The `IOB1` default maintains
+        `Conll2003DatasetReader` assumes a coding scheme of input data is `IOB1`.
+        Valid options are `None` and `BIOUL`.  The `None` default maintains
```

Comment:
https://github.com/allenai/allennlp/pull/4924#discussion_r566305479

@epwalsh 
I added a note about the assumed coding scheme.
Does this change match your intention?